r_examples/rstudio_sagemaker_pipelines_and_serving/main.R

```diff
@@ -0,0 +1,84 @@
+# --------- Make sure you have all the necessary packages installed ------------------
+library(dplyr)
+library(reticulate)
+if (!py_module_available("sagemaker-studio-image-build")){py_install("sagemaker-studio-image-build", pip=TRUE)}
+library(readr)
+
+sagemaker <- import('sagemaker')
+boto3 <- import('boto3')
+
+session <- sagemaker$Session()
+bucket <- session$default_bucket()
+
+role_arn <- sagemaker$get_execution_role()
+account_id <- boto3$client("sts")$get_caller_identity()$"Account"
+region <- boto3$session$Session()$region_name
+
+# --------- Build the containers ---------
+
+local_path <- dirname(rstudioapi::getSourceEditorContext()$path)
+system(paste0("cd ", local_path, " ; sm-docker build . --file ./docker/Dockerfile-processing --repository sagemaker-r-processing:1.0"))
+
+system(paste0("cd ", local_path, " ; sm-docker build . --file ./docker/Dockerfile-train-n-deploy --repository sagemaker-r-train-and-deploy:1.0"))
+
+# --------- Get data ---------
+data_file <- 'http://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data'
```

Comment:
Can you use the AWS-hosted version in S3?
`s3://sagemaker-sample-files/datasets/tabular/uci_abalone/abalone.csv`

Reply:
good point, updated